packages/panels/src/components/sidebar-menu/SidebarMenuCollapsible.tsx

```diff
@@ -1,53 +1,104 @@
 import * as React from "react";
 import { ReactNode, useState } from "react";
-import { Collapsible } from "../collapsible/Collapsible";
-import { Box, Column, Indent } from "@stenajs-webui/core";
+import { Box, Indent, Row, Space, Text } from "@stenajs-webui/core";
 import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
-import styles from "./SidebarMenuCollapsible.module.css";
 import { Icon } from "@stenajs-webui/elements";
+import contentStyles from "./SidebarMenuContent.module.css";
+import styles from "./SidebarMenuCollapsible.module.css";
+import cx from "classnames";
+import { faChevronUp } from "@fortawesome/free-solid-svg-icons/faChevronUp";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+import { cssColor } from "@stenajs-webui/theme";
+import { SidebarMenuCollapsibleGroupBox } from "./SidebarMenuCollapsibleGroupBox";
 
 export interface SidebarMenuCollapsibleProps {
   label: string;
   collapsed?: boolean;
   leftIcon?: IconDefinition;
   children?: ReactNode;
+  className?: string;
+  initialExpand?: boolean;
+  indent?: number | boolean;
 }
 
 export const SidebarMenuCollapsible: React.FC<SidebarMenuCollapsibleProps> = ({
   children,
   label,
   leftIcon,
+  className,
+  initialExpand = false,
+  indent,
 }) => {
-  const [collapsed, setCollapsed] = useState<boolean>(false);
+  const [expanded, setExpanded] = useState<boolean>(initialExpand);
+
+  const innerClassName = cx(
+    contentStyles.button,
+    expanded ? contentStyles.selected : undefined,
+    className
+  );
+
+  const innerStyle = {
+    height: "var(--swui-sidebar-menu-item-height)",
+  };
 
   return (
-    <Box background={"var(--current-background-color)"}>
-      <Collapsible
-        className={styles.sidebarMenuCollapsible}
-        label={label}
-        collapsed={collapsed}
-        onClick={() => setCollapsed(!collapsed)}
-        contentLeft={
-          leftIcon ? (
-            <Box
-              width={"var(--swui-sidebar-menu-item-height)"}
-              alignItems={"center"}
-              justifyContent={"center"}
-            >
+    <Box
+      background={"var(--current-background-color)"}
+      aria-expanded={expanded}
+    >
+      <Box
```

Comment:
Collapsible gone 🥲

Reply:
Yes. This might be temporary, we will redesign collapsible next, and if they match up nicely, we will use it again.